pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -260,6 +260,18 @@ self: super: {
       };
     });
 
+  unisonSyntax = buildVimPluginFrom2Nix {
+    pname = "unison-syntax";
+    version = "2019-08-21";
+    src = fetchFromGitHub {
+      owner = "unisonweb";
+      repo = "unison";
+      rev = "08effd4535d2f388d9548ed238755bb38d48175b";
+      sha256 = "0kbx5h41hsqzmfq6kb2c3p3a59z0xjvwg6ixyv4gs0kwkj7bvdfj";
+    };
+    preInstall = "cd editor-support/vim";
```

Comment:
This should be `postPatch`. It doesn't make a practical difference here as long as the build phase is empty, but it makes more sense semantically.

Reply:
Ah thanks, and thanks for the review!  I'll swing back to this shortly and incorporate your suggestions.  Thanks!